dashboard/serializers_test.py

```diff
@@ -151,22 +161,28 @@ def setUpTestData(cls):
             UserProgramSearchSerializer.serialize_enrollments(fa_mmtrack, cls.fa_enrollments)
         )
         cls.fa_semester_enrollments = UserProgramSearchSerializer.serialize_semester_enrollments(cls.fa_enrollments)
+        cls.fa_final_grades = UserProgramSearchSerializer.serialize_final_grades(fa_mmtrack, cls.fa_enrollments)
 
     def test_full_program_user_serialization(self):
         """
         Tests that full ProgramEnrollment serialization works as expected
         """
         self.profile.refresh_from_db()
         program = self.program_enrollment.program
-        assert UserProgramSearchSerializer.serialize(self.program_enrollment) == {
+        expected_values = {
             'id': program.id,
             'enrollments': self.serialized_enrollments,
             'semester_enrollments': self.semester_enrollments,
+            'final_grades': self.final_grades,
             'grade_average': 75,
             'is_learner': True,
             'num_courses_passed': 1,
             'total_courses': 1
         }
+        sorted_serialized_enrollments = self.get_sorted_final_grades_values(
+            UserProgramSearchSerializer.serialize(self.program_enrollment)
+        )
+        assert sorted_serialized_enrollments == self.get_sorted_final_grades_values(expected_values)
```

Comment:
This looks like we lost the assertion of properties other than final_grades on the serializer return value here and on the next two tests.